sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -213,6 +200,19 @@ public partial class MetricsAdvisorKeyCredential
         public void UpdateApiKey(string apiKey) { }
         public void UpdateSubscriptionKey(string subscriptionKey) { }
     }
+    public static partial class MicrosoftAzureMetricsAdvisorRestAPIOpenAPIV2ModelFactory
```

Comment:
hmm is this the model factory? why that name?

Reply:
I think it comes from the `info.title` property in the swagger: https://github.com/Azure/azure-rest-api-specs/blob/08f5e391f2153a99580b458cc71ef88e45dd0531/specification/cognitiveservices/data-plane/MetricsAdvisor/preview/v1.0/MetricsAdvisor.json#L4